### ISSUE TITLE
aarch64 build?

### ISSUE THREAD

**Issue by JamiKettunen** [flathub/io.github.zen_browser.zen#52]
With the likes of the surprisingly powerful (matching my current workstation with Ryzen 9 5950X in compile speeds!) Snapdragon X Elite laptops etc running Linux via https://discourse.ubuntu.com/t/ubuntu-24-10-concept-snapdragon-x-elite/48800 among others if not already in the coming months it'd be nice to have this built and available for aarch64 architecture as well similar to https://flathub.org/apps/io.gitlab.librewolf-community and https://flathub.org/apps/one.ablaze.floorp (even though interestingly https://flathub.org/apps/org.mozilla.firefox itself isn't yet either) :)

I'm getting an HP OmniBook X 14 next week so I can help with testing once I'm all setup on Linux with it, maybe I'll even adapt the `flatpak-builder` yml myself ^^

**Comment by JamiKettunen** [comment]
Ah, seems Zen browser itself isn't being built for aarch64 windows/linux in general so this would be blocked on https://github.com/zen-browser/desktop/issues/1051 (tho for Linux as the issue is kinda about WoA)

**Comment by NN708** [comment]
The Aarch64 build is now available. Will it be added to Flathub?

**Comment by mauro-balades** [comment]
Should be done now
